src/libs/actions/BankAccounts.js

```diff
@@ -627,6 +627,16 @@ function hideBankAccountErrorModal() {
     Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {isErrorModalVisible: false});
 }
 
+/**
+ * Set the current fields with errors.
+ *
+ * @param {String} errors
+ */
+function setBankAccountFormValidationErrors(errors) {
+    Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {errors: null}); // TOOD: How to force removal
+    Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {errors});
```

Comment:
Just out of curiosity here... did you figure out that if we merge errors with `null` and then an object that it will successfully overwrite the key? I wonder then if it would be better to have something like 

```js
Onyx.setProperty(ONYXKEYS.REIMBURSEMENT_ACCOUNT, 'errors', errors);
```

I'm not sure if we want to use the workaround or fix this in Onyx (I'm thinking fix it in Onyx would be better) but if we do leave in the workaround we should add some comments about why we are doing it and probably create an issue to fix it later instead of leaving a `TODO`.

Reply:
The workaround of setting to `null` and then the actual objects has been working so far, but as you say, it is a workaround.

I think it would be great to fix Onyx, much better than having this kind of workaround. I was just being lazy and postponing it because I haven't touch that repository and I don't know if that is going to branch out into something complex... but maybe it is an easy change? no idea.

Reply:
Ok cool. No worries. That's a good instinct as it could be a larger change. Let's ask in `#engineering-chat` tomorrow to see if anyone has ideas.

And worst case scenario we can just add a code comment that it is a workaround + create an issue to fix it when we are ready.

Reply:
Looking at the `merge` function in `Onyx` and the function `applyMerge` [here](https://github.com/Expensify/react-native-onyx/blob/6eadd7f73fca87e6551e43607ec78f645e17ef50/lib/Onyx.js#L548-L550), I think it deletes keys that we set values `null` or `undefined`. You could make very simplistic implementations of the following functions:

```
function removeProperty(key, property) {
   merge(key, { [property]: null });
}

function setProperty(key, property, value) {
   removeProperty(key, property);
   merge(key, { [property]: value });
}
```

If performance is an issue, I guess it could be worth doing something more complex.

Reply:
Interesting idea. I started a thread here -> https://expensify.slack.com/archives/C01GTK53T8Q/p1630692676411000

My gut says that relying on the `merge()` behavior could be risky in case someone decides it should work differently one day then the `removeProperty` and `setProperty` methods would also break unexpectedly (but we could write unit tests to hopefully prevent that).

Reply:
Agreed, because `merge` clearing a key when the value is `null` is not really part of the documentation of the API, so it could change, and also agree that unit test could offset the risk.

Reply:
Created a follow up issue here and will look into a better solution

https://github.com/Expensify/Expensify/issues/176419